schemas/2016-02-03/Microsoft.Devices.json

```diff
@@ -355,6 +370,32 @@
         }
       },
       "description": "The properties of the Storage Endpoint provided by the customer for file upload"
+    },
+    "IpFilterRule": {
+      "description": "IP filter Rule",
+      "type": "object",
+      "properties": {
+        "filterName": {
```

Comment:
I don't understand what it means to add expression support. Can you please explain?